app/views/govuk_publishing_components/components/docs/layout_super_navigation.yml

```diff
@@ -0,0 +1,37 @@
+name: Super navigation header
```

Comment:
I've update the name to start with header - not that I'm particularly happy with the name in the first place to be honest! But since it should only be used in Static and in the public layout component, it wouldn't be too hard to rename it.

https://github.com/alphagov/govuk_publishing_components/blob/951b14bb286ae910bb3104b2ce61dd017f910eae/app/views/govuk_publishing_components/components/docs/layout_super_navigation_header.yml#L1